config_template.cfg

```diff
@@ -1943,3 +2461,39 @@ MAX_BASIS_DIM = 100
 %
 % Frequency of snapshots saves, for unsteady problems (default: 1. 2 means every other)
 ROM_SAVE_FREQ = 1
+
+% --------------------- PASTIX PARAMETERS -----------------------%
+%
+% Number of calls to 'Build' that trigger re-factorization (0 means only once)
+PASTIX_FACTORIZATION_FREQUENCY= 1
+%
+% 0 - Quiet, 1 - During factorization and cleanup, 2 - Even more detail
+PASTIX_VERBOSITY_LEVEL= 0
+%
+% Level of fill for PaStiX incomplete LU factorization
+PASTIX_FILL_LEVEL= 1
+
+% --------------------- PYTHON PARAMETERS -----------------------%
+% DESCRIPTION: Gradient method
+%GRADIENT_METHOD
+%
+% DESCRIPTION: Geometrical Parameter
+%GEO_PARAM
+%
+% DESCRIPTION: Current value of the design variables
+%DV_VALUE_NEW
+%
+% DESCRIPTION: Previous value of the design variables
+%DV_VALUE_OLD
+%
+% DESCRIPTION: Number of partitions of the mesh
+NUMBER_PART= 1
+%
+% DESCRIPTION: Verbosity of the python scripts to Stdout
+%CONSOLE
+%
+% DESCRIPTION: Flag specifying if the mesh was decomposed
+%DECOMPOSED
+%
+% DESCRIPTION: Number of zones of the problem
+%NZONES= 1
```

Comment:
These are not supposed to be set by users

Reply:
OK, I will remove these then.